src/pages/home/report/ReportActionItem.js

```diff
@@ -232,6 +234,12 @@ class ReportActionItem extends Component {
 
         const reactions = _.get(this.props, ['action', 'message', 0, 'reactions'], []);
         const hasReactions = reactions.length > 0;
+        const shouldDisplayThreadReplies = lodashGet(this.props.action, 'childReportID', 0) !== 0 && lodashGet(this.props.action, 'childCommenterCount', 0) > 0;
+
+        let oldestFourEmails = lodashGet(this.props.action, 'childOldestFourEmails', []);
+        if (!_.isArray(oldestFourEmails)) {
+            oldestFourEmails = oldestFourEmails.split(',');
+        }
```

Comment:
@chiragsalian is there a Util function or something for this

Reply:
I'm a little confused by this. We're adding a new param `oldestFourEmails` right? So why wouldn't we know the type at this stage?

Reply:
Sorry util function for what specifically? If you are asking to convert string to array then no but your split looks fine to me. you can add a comment saying its temporary if you want. I'll be investigating its improvements [here](https://github.com/Expensify/Expensify/issues/282697).

Reply:
> So why wouldn't we know the type at this stage?

Well Georgia told me its sometimes an array and sometimes a string. Now that i think about it I'm not sure why it would change. From PHP layer i only send this as a string atm. Maybe @grgia can shed some light on when this is an array.

P.S i can update the backend code to only send array since it looks like that's what we prefer. I'll make that change soon.

Reply:
also if we know its always string this line could be simplified like so,
```
const oldestFourEmails = lodashGet(this.props.action, 'childOldestFourEmails', '').split(',');
```

Reply:
Oh I just meant that other similar values are an array format (like participants) so this piece of code was from waiting on confirmation on the shape of `childOldestFourEmails` but now we know it's a string I'll update!